ffi/rust/src/ffi_wrapper.go

```diff
@@ -0,0 +1,97 @@
+package main
```

Comment:
I'm not sure where this file should sit. It's not really unique to FFI, it could and should be used for all future FFIs. Maybe we can have it in `ffi/` ?